deploy/dependencies/prometheus-operator-cluster-role-binding.yaml

```diff
@@ -10,4 +10,4 @@ roleRef:
 subjects:
   - kind: ServiceAccount
     name: monitoring-stack-operator-prometheus-operator-sa
-    namespace: default
+    namespace: operators
```

Comment:
why is this change needed 🤔? 
Should this be a separate pr unrelated to deletion of Grafana?

Reply:
Indeed this can come in a separate PR